build_projects/update-dependencies/UpdateFilesTargets.cs

```diff
@@ -87,25 +87,17 @@ public static BuildTargetResult ReplaceProjectJson(BuildTargetContext c)
         {
             List<DependencyInfo> dependencyInfos = c.GetDependencyInfos();
 
-            string[] sourceDirsToUpdate =
+            var projectJsonFiles = new List<string>
```

Comment:
I thought we had files like DependencyVersion.cs that were updated by Maestro and will no longer be updated. Is that correct?